conf/scores.d/rbl_group.conf

```diff
@@ -29,15 +29,15 @@ symbols = {
         description = "Sender listed at https://www.dnswl.org, no trust";
     }
     "RCVD_IN_DNSWL_LOW" {
-        weight = 0.0;
+        weight = -0.5;
         description = "Sender listed at https://www.dnswl.org, low trust";
     }
     "RCVD_IN_DNSWL_MED" {
-        weight = 0.0;
+        weight = -2.0;
         description = "Sender listed at https://www.dnswl.org, medium trust";
     }
     "RCVD_IN_DNSWL_HI" {
-        weight = 0.0;
+        weight = -4.0;
```

Comment:
I am afraid this would make things more complicated, especially when it comes to mailing lists (the OpenBSD list, for example, is listed as "high" in DNSWL, and quite an amount of mails there do not have a DKIM signature).

Talking about score values, I could also settle on -0.5/-1.0/-2.0. That way, we have some whitelisting effect here, but not too much.

Reply:
If you want to whitelist maillists, you can use the force actions module and set the `MAILLIST` symbol to `no action`.